hazelcast-jet-distribution/src/root/release_notes.txt

```diff
@@ -37,16 +37,17 @@ Thank you for your valuable contributions!
 
 1. New Features
 
-2. Enhancements
+[jdbc] Implement configurable batch size limit in JDBC sink processor (#2888).
 
-[extensions] Ensure naming conventions for sources/sinks (#2685)
+2. Enhancements
 
 3. Fixes
 
-[core] Jet now doesn't close System.out during JVM shutdown (#2649)
-
 4. Breaking Changes
 
+[jdbc] The `SinkProcessors.writeJdbcP()` and `WriteJdbcP.metaSupplier()` methods
+ gained a new argument: `batchLimit`.
```

Comment:
`WriteJdbcP.metaSupplier` is private API:
```suggestion
[jdbc] Added `batchLimit` parameter to `SinkProcessors.writeJdbcP()`  method.
```

Reply:
> WriteJdbcP.metaSupplier is private API.

Oh right, is an internal implementation class!